elements/bulbs-poll/components/answers.js

```diff
@@ -1,16 +1,24 @@
 import React, { PropTypes } from 'react';
 import Answer from './answer';
+import ImageAnswer from './image-answer';
 import classnames from 'classnames';
 
 export default function Answers (props) {
-  let classNames = classnames('bulbs-poll-answers', {
+  let AnswerType = Answer;
+  let ListType = 'bulbs-poll-answers';
```

Comment:
`AnswerType` needed to be capitalized because react requires `<Component>` to be capitalized as a constructor.

Something like `listClassName` would be more descriptive here.